tests/debugging/test_debugger.py

```diff
@@ -733,6 +676,68 @@ def count_status(queue):
         RemoteConfigClient.request = old_request
 
 
+def test_probe_status_logging_reemit_on_modify(monkeypatch):
```

Comment:
I think we should also add a new test a modified probe (or update an existing test):
1. get the right version inside the snapshot json event
2. update log template and the new event get updated log template and version

WDYT?